redaxo/src/addons/be_style/scss/_mime.scss

```diff
@@ -1,6 +1,7 @@
 .rex-mime {
     @extend .fa;
-    @extend .fa-file-o;
+    @extend .fa-regular;
+    @extend .fa-file;
     position: relative;
     font-size: 3em;
 }
```

Comment:
Du müsstest bitte prüfen, dass `.rex-mime` im kompilierten CSS vor allen anderen `.rex-mime-`-Icons ausgegeben wird. Ansonsten werden, via @gharlan [hier](https://github.com/redaxo/redaxo/pull/5712#issuecomment-1657208146) angemerkt, keine Mime Icons im Medienpool dargestellt..

Reply:
Soweit ich das sehe, stimmt die Reihenfolge. Es wird jedoch – wie korrekt angemerkt – das default-Icon geladen. Woran könnte es noch liegen?

Reply:
Da war ich zu ungenau. Im **Scss** steht `.rex-mime` weiter oben als alle anderen. Durch den Einsatz von `@extend` landet aber `.rex-mime` im kompilierten **CSS** aber nach den spezifischen `.rex-mime-...`

Reply:
Uff, und wie löse ich das? Was muss ich anpassen? Ich kenne mich mit dem Kompilieren nicht aus. Wo müsste ich die Reihenfolge ändern, damit es richtig rauskommt?

Reply:
Du kannst es nur beeinflussen, wenn du auf das @extend verzichtet. Der Compiler sammelt alle Klassen ein und gibt diese dann irgendwas irgendwo im CSS aus. Wie das genau funktioniert kann ich dir nicht sagen.
Daher würde ich hier generell auf die @extend Anweisung verzichten und eher das Icon direkt notieren. Das erreichst du so u.a.

**Alt**
```
.rex-mime-pdf {
    @extend .fa-file-pdf;
}
```

** Neu**
```
.rex-mime-pdf::before {
    content: unquote("\"#{ $fa-var-file-pdf }\"");
}
```

Eventuell wäre es sinnvoller, alle Icons in ein Array zu packen und dann via Schleife abzuarbeiten.
Wie hier bei FA zum Beispiel:
https://github.com/redaxo/redaxo/blob/6e86b5ae5be6512acb73739967487b9dffff8dee/redaxo/src/addons/be_style/vendor/font-awesome/scss/_icons.scss#L7-L9